clang/test/SemaSYCL/Inputs/CL/sycl/detail/kernel_desc.hpp

```diff
@@ -43,8 +43,6 @@ __SYCL_INLINE_NAMESPACE(cl) {
     }
     static constexpr const char *getName() { return ""; }
     static constexpr bool isESIMD() { return 0; }
-    static constexpr bool callsThisItem() { return false; }
```

Comment:
Can you also please check  `clang/test/CodeGenSYCL/Inputs/sycl.hpp`. I'm not sure but there might be code to be removed there as well